zerver/views/push_notifications.py

```diff
@@ -66,3 +67,25 @@ def remove_android_reg_id(
     validate_token(token, PushDeviceToken.GCM)
     remove_push_device_token(user_profile, token, PushDeviceToken.GCM)
     return json_success(request)
+
+
+@human_users_only
+@has_request_variables
+def send_test_push_notification_api(
+    request: HttpRequest, user_profile: UserProfile, token: str = REQ()
```

Comment:
Hmm, would it make sense to do a `default=None` thing for `token`? I feel like that's how we usually handle optional parameters...